packages/main/src/components/VariantManagement/ManageViewsDialog.tsx

```diff
@@ -32,6 +32,14 @@ addCustomCSSWithScoping(
   }
   `
 );
+addCustomCSSWithScoping(
+  'ui5-dialog',
+  `
+  :host([data-component-name="VariantManagementManageViewsDialog"]) .ui5-popup-footer-root{
+    padding: 0;
+  }
+  `
+);
```

Comment:
Please use the [::part pseudo element](https://developer.mozilla.org/en-US/docs/Web/CSS/::part) instead for both injected styles. 
--> remove `addCustomCSSWithScoping` for the content and the footer and add the styles with `createUseStyles` 
E.g.: 
```
   <className of Dialog>: {
      '&::part(content)': {
        padding: 0
      },
      '&::part(footer)': {
        padding: 0
      }
    },
```